interface/main/main_title.php

```diff
@@ -66,25 +66,32 @@ function showhideMenu() {
 		document.getElementById("showMenuLink").innerHTML = '<?php echo htmlspecialchars( xl('Show Menu'), ENT_QUOTES); ?>';
 	}
 }
+
+function prepareLogosStyle(){
+	var left_cell = document.getElementsByClassName("left_cell")[0];
+	left_cell.style.cssText = 'padding-left: 110px !important';
+	document.getElementById("tinylogocontainer").style.position = "absolute";
+}
```

Comment:
If I understood your comment correctly - 
Because the main_title html uses a 'table' I was not able to find css that can fit for both when logos are turned on and when logos are turned off. I tried for a while. That's why I has to use scripting to change the css when logos are turned on. Would you like for this js to be placed somewhere else?
And a question: I was wondering if there is js file and a css file that is used in ALL themes. 
Thank you.